onboarding/settings/common.py

```diff
@@ -0,0 +1,2 @@
+def plugin_settings(settings):
```

Comment:
Common settings file by default has other Django settings. Do we need them here?

Reply:
File deleted